aea/protocols/generator/base.py

```diff
@@ -1948,10 +1968,18 @@ def generate_protobuf_only_mode(self, run_protolint: bool = True) -> None:
                 )
             )
 
-        if not is_valid_protobuf_schema:
-            # Remove the generated folder and files
-            shutil.rmtree(output_folder)
-            raise SyntaxError("Error in the protocol buffer schema code:\n" + msg)
+        # Run black formatting
+        try_run_black_formatting(self.path_to_generated_protocol_package)
+
+        # Run isort formatting
+        try_run_isort_formatting(self.path_to_generated_protocol_package)
```

Comment:
Why would we run this as part of the protobuf only mode?

Reply:
For the `*_pb2.py` file

Reply:
Should only run for the python ones though.